var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -40,13 +40,17 @@ class Lbann(CMakePackage):
     variant('gpu', default=False, description='Builds with support for GPUs via CUDA and cuDNN')
     variant('opencv', default=True, description='Builds with support for image processing routines with OpenCV')
     variant('seq_init', default=False, description='Force serial initialization of weight matrices.')
+    variant('el_cublas', default=False, description='Builds with support for cuBLAS offload in Elemental')
```

Comment:
I don't think this is the right way to do this. If you want to get the same functionality, build:
```
$ spack install lbann ^elemental+cublas
```
You can replace `'+el_cublas' in spec` with `'^elemental+cublas' in spec` below.